File: firstcode/firstassignemt/newassignement/divideorsquare.py
import math
def divide_or_square(number):
	if number % 5 == 0:
		return math.sqrt(number)
	elif number % 5 != 0:
		return number % 5 

number = int(input("Enter a number:  "))
result = divide_or_square(number)
print(result)

File: firstcode/firstassignemt/myjavaassignment/myfunctions/DivideOrSquare.java
import java.util.Scanner;

public class DivideOrSquare {
	public static void main(String[] args) {
		System.out.println("Welcome friend, it is a good day");
		
	Scanner input = new Scanner(System.in);
		
	System.out.print("Enter a number:  ");
	int number = input.nextInt();
	
	CommonTools tools = new CommonTools();
	
	double result = tools.divideorsquare(number);
	
	System.out.printf("%.2f ", result);

}
}